backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/visitor/impl/element/InputOutputVisitor.java

```diff
@@ -23,7 +22,7 @@ public boolean canBeTransformed(DomElementVisitorContext context) {
   }
 
   @Override
-  protected Severity messageSeverity() {
-    return Severity.INFO;
+  protected boolean isSilent() {
```

Comment:
Should be commented that it supress the result of `visitCamundaElement()`. 

This is not obvious.